Comparing /root/./workspace/apache/flink/384691544/.m2/org/apache/flink/flink-java/1.1.4/flink-java-1.1.4.jar with /root/workspace/apache/flink/384691544/flink-java/target/flink-java-1.3-SNAPSHOT.jar:
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.java.ExecutionEnvironment  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) int getDefaultLocalParallelism()
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.java.operators.Operator  (not serializable)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.operators.ResourceSpec getMinResources()
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.operators.ResourceSpec getPreferredResources()
***  MODIFIED CLASS: PUBLIC org.apache.flink.api.java.utils.ParameterTool  (compatible)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.flink.api.java.utils.ParameterTool fromPropertiesFile(java.io.File)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.flink.api.java.utils.ParameterTool fromPropertiesFile(java.io.InputStream)
